### ISSUE TITLE
class table on slot pages

### ISSUE THREAD

**Issue by sujaypatil96** [linkml/linkml#1126]
Feature request:

Add a table to slot pages similar to the slots table on class pages. For example, https://microbiomedata.github.io/nmdc-schema/Biosample/#slots

The new table for slot pages should include the same columns as the above slots table, i.e.,
* name
* range and cardinality
* description
* inheritance

The only variable column in the above table will be the `name` field. In the above example table, you can see slot names. In the table for slot pages, we should see class names.

The table this issue is requesting should be called **slot modifying classes**. As the name indicates, we want to show all those classes that modify the slot under consideration within their `slot_usage` property.

The way to solve this issue would be to loop through all classes, look at their `slot_usage` property, and if you see the slot under consideration under there, then show it in the _slot modifying classes_ table.

CC: @putmantime

**Comment by amc-corey-cox** [comment]
Sorry for the delay on this. I'm starting work on this issue now. Please let me know if you have further thoughts.

**Comment by amc-corey-cox** [comment]
I have submitted to PR's that will close this issue. I have encountered some issues with tests failing that I stated my workarounds for in the appropriate PR's. Please review these when you get a chance.

**Comment by sujaypatil96** [comment]
Thank you for working on PR #1211 @amc-corey-cox. I had an additional request for you if you don't mind.

So what you made the PR for, based on my request was a table to show classes that modify certain slots in a table called Slot modifying Classes which makes perfect sense. However, I was thinking that it would be better to combine that information with another table that we are already creating called Applicable Classes. So instead of having another table called Slot modifying Classes do you think it might be better to combine the two and add the slot modification information as a column to the Applicable Classes table?

So basically what the Applicable Classes table would look like is:
Name      | Description  | Slot Modified in Class  |
--------- | ------------ | ----------------------- |
Biosample |  random      | yes or no               |
